src/containers/CheckoutPage/CheckoutPage.module.css

```diff
@@ -243,7 +243,7 @@
   color: var(--colorFail);
 }
 .notFoundError {
-  color: var(--colorAttention);
+  color: var(--colorAttentionDark);
```

Comment:
This is strange. I'm not sure why error text is yellow in the first place. I.e. on purpose or is this some kind of copy-paste bug. 

No need to change this to red on this PR, but perhaps it's worth testing this error during some cooldown week.